### DISCUSSION TITLE
Authentication Failure The Key file provided either does not exist or is not accessible. Please check the path to the file. Error: Key file not found

### DISCUSSION THREAD

**Discussion by leandrofelix-dev** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#439] [Q&A]
Eu tenho este erro ao instalar o aplicativo. Can someone help me?
![Sem título](https://user-images.githubusercontent.com/58048297/124353888-98a3e580-dbdf-11eb-929e-3ba30beb1f65.png)
![Sem título2](https://user-images.githubusercontent.com/58048297/124353891-9b9ed600-dbdf-11eb-918d-9a0f5ab811fa.png)

**Comment by sumeshir26** [comment]
Hi @lehinfo-felix it seems like the file path that you set does not exist(You might have deleted it or moved it) or requires admin privileges.
Double-check your path and click the 'Browse' button to select one.
